tock/api/urls.py

```diff
@@ -19,7 +19,11 @@
         Submissions.as_view(),
         name='Submissions'
     ),
+    # Note that this endpoint returns TimecardObjects
     path('timecards.json', TimecardList.as_view(), name='TimecardList'),
+    # ...and this one returns Timecards (referred to as "full timecards" to disambiguate
+    # from existing endpoints / naming conventions)
+    path('full_timecards.json', FullTimecardList.as_view(), name='FullTimecardList'),
```

Comment:
I'm very open to naming conversations here, wanted to put this CR up first and start the conversation before going on any refactoring voyage to rename "timecard"-related things that are in fact `TimecardObject`s

Reply:
I'm drawn to embarking on a renaming voyage as well but this will suffice for our immediate. The additional comments definitely help here.